dm/master/server.go

```diff
@@ -324,7 +324,7 @@ func (s *Server) OperateTask(ctx context.Context, req *pb.OperateTaskRequest) (*
 	var wg sync.WaitGroup
 	for _, worker := range workers {
 		// https://golang.org/doc/faq#closures_and_goroutines
-		worker := worker
+		worker1 := worker
```

Comment:
🤔 I think you can use `worker1` in the closures, and no need these two lines.

Reply:
:joy:  Yes, I omit that before. Revised